### ISSUE TITLE
dynamic module second import still slowly when has import `std@0.56.0/http/server.ts`

### ISSUE THREAD

**Issue by shynome** [denoland/deno#6226]
### deno version
debian 10
```
deno 1.0.5
v8 8.4.300
typescript 3.9.2
```

### report steps

you need `wrk` or other tools for test

- `git clone https://github.com/shynome/deno-report-dynamic-module-exec-so-slowly.git`
- `cd deno-report-dynamic-module-exec-so-slowly`
- `deno run -A server.ts`
- `curl http://127.0.0.1:8000/b` work fine
- `curl http://127.0.0.1:8000/c` work fine
- `wrk http://127.0.0.1:8000/b` my qps test result is 79
- `wrk http://127.0.0.1:8000/c` my qps test result is  6389

### difference
the difference is `b.ts` has imported type `import type { ServerRequest } from "https://deno.land/std@0.56.0/http/server.ts"`, if I remove this line, `b.ts` will work fine

```ts
//b.ts
import type { ServerRequest } from "https://deno.land/std@0.56.0/http/server.ts";
export default (req: ServerRequest) => {
  req.respond({ body: "bbbb" });
};
```

```ts
//c.ts
//import type { ServerRequest } from "https://deno.land/std@0.56.0/http/server.ts";
export default (req: any) => {
  req.respond({ body: "bbbb" });
};
```

**Comment by shynome** [comment]
I have update reproudce repo, the new repo has less code and make test easier

**Comment by shynome** [comment]
When I add a simple module cache to the code, both qps are close to 20k in my PC

I think "import" should have a cache, but I don't know the `import` is 
whether equally slow in the browser

**Comment by shynome** [comment]
node dynamic import is also heavye, but deno dynamic import is look like have ignore the compiled files and recompile everytime

**Comment by bartlomieju** [comment]
@shynome double compilation problem was fixed in #6177. Closing as resolved, feel free to reopen if you're still experiencing problems.
